### ISSUE TITLE
Does it work on L1 Device?

### ISSUE THREAD

**Issue by MoHa-0** [aidanmacgregor/Netflix_ATV_L3_DRM_Uncertified_Mod#10]
Hi @aidanmacgregor , recently I discovered Your work with netflix for uncertified devices, so I installed it on my android box (L3), and working perfectly, but when i install it on my tv(L1) shows me ui 800 3. Is That because my tv is widevine L1, and if it is,is there any way to fix it, or a different to watch netflix on it?
(Neither Android Box or Android TV are not certified by Netflix)
Thanks in advance!

**Comment by the4anoni** [comment]
https://github.com/longseespace/netflix_patch

Try this maybe it works

**Comment by aidanmacgregor** [comment]
Seems some L1 devices work (only SD) like my Samsung phones, i dont have any android TV L1 devices to test with though but others repot success with L1 :)

**Comment by tuncaygunes** [comment]
Is this apk only for ATV devices? Or will it work with AOSP devices too?

**Comment by aidanmacgregor** [comment]
> Is this apk only for ATV devices? Or will it work with AOSP devices too?

It should work on AOSP too, works on several of my phones :)

**Comment by grega74** [comment]
Hello there ...

I have L1 capable box but no official support for Netflix. you verson works ok, but only SD (since it's L3)... is it possible to make apk that would work HD and 4K on my machine? Regards

**Comment by aidanmacgregor** [comment]
> Hello there ...
> 
> I have L1 capable box but no official support for Netflix. you verson works ok, but only SD (since it's L3)... is it possible to make apk that would work HD and 4K on my machine? Regards

no, there have been several methods over the years but they don't work anymore and required everyone to mod their app individually
